test/project_deno/gleam.toml

```diff
@@ -0,0 +1,16 @@
+name = "project"
+version = "0.1.0"
+target = "javascript"
+
+[javascript]
+typescript_declarations = true
+runtime = "deno"
+
+[javascript.deno]
+location = "http://localhost:8080"
```

Comment:
This doesn't seem to get used anywhere!

Reply:
Just pushed some tests to assert location API is defined correctly via this config option.